src/main/native/jni_utils.h

```diff
@@ -0,0 +1,13 @@
+#ifndef _Included_org_bblfsh_client_libuast_Libuast_jni_utils
+#define _Included_org_bblfsh_client_libuast_Libuast_jni_utils
+
+#include <jni.h>
+
+extern const char *CLS_NODE;
+extern const char *CLS_CTX;
+
+JNIEnv *getJNIEnv();
+jobject NewJavaObject(JNIEnv *, const char *, const char *, ...);
```

Comment:
I'm not sure I follow the naming convention in use here. [Google style](https://google.github.io/styleguide/cppguide.html#Function_Names) is `MixedCaseLeadingCaps` for classes and functions (with a carve-out for class accessors to follow `snake_case` style). But we seem to be mixing that and conventional camel.

(I'm not wedded to Google style, I only mentioned that because you included a clang-format config based on that style)

Reply:
This is an old code, I belive it follow the same that was used in [libuast](https://github.com/bblfsh/libuast/blob/master/src/uast.h).

I'm not settled on this one either, but I think it's totally fine to use almost whatever in a the well-localized JNI wrappers themselves.

Will be happy to refactor it, if needed, and add a better formatter config to all the projects later on, as soon as we have something functional here.  I'm mostly not very worried about it here because of the limited scope and no expectations for this native piece to grow.

TBH Google one from above was just placed randomly, to have at least something, and mostly because of 2-space indentation.

Reply:
Yeah, this doesn't need to be addressed right now, I just wanted to make a note of it while we're paying attention to this code. 🙂